tools/elevator.cpp

```diff
@@ -0,0 +1,44 @@
+#include <Windows.h>
+#include <iostream>
+#include <string>
+#include <vector>
+
+int main(int argc, char *argv[]) {
```

Comment:
A doxygen comment would be good to add to the main function

https://docs.lizardbyte.dev/projects/sunshine/en/nightly/source/src.html#example-documentation-blocks